website/assets/apps/megamap/helpers/Cartographer.js

```diff
@@ -358,7 +362,8 @@ export default class Cartographer {
     const totalCostRow = document.createElement('div');
     totalCostRow.setAttribute('class', 'popup-row');
     const totalCostLabelDiv = document.createElement('div');
-    totalCostLabelDiv.appendChild(document.createTextNode('Total Reported Cost'.toUpperCase()));
+    totalCostLabelDiv.setAttribute('class', 'popup-row-header');
+    totalCostLabelDiv.appendChild(document.createTextNode('Total Reported Cost'));
```

Comment:
This might be a question for RA, but Page 7 has a different set of rows from our implementation:
* Spec: Locations, Type, Status
* Coded: Locations, Type, Total Reported Cost

Which one is correct?